PhpAmqpLib/Connection/AMQPConnectionConfig.php

```diff
@@ -138,6 +138,15 @@ final class AMQPConnectionConfig
      */
     private $debugPackets = false;
 
+
+    /**
+     * @var int|null $socketProtocolMode Parameter specifies the protocol family to be used by the socket.<br>
+     *    Const AF_INET IPv4 Internet based protocols.<br>
+     *    Const AF_INET6 IPv6 Internet based protocols.<br>
+     *    null Auto select
+     */
+    private $socketProtocolMode = null;
```

Comment:
that's probably simply `socketProtocol`? word mode is redundant here

Reply:
this is the mode because if you do not install the protocol, automatic will be selected